helm-charts/splunk-otel-collector/templates/config/_otel-agent.tpl

```diff
@@ -40,6 +40,7 @@ receivers:
   {{- if (eq (include "splunk-otel-collector.metricsEnabled" .) "true") }}
   hostmetrics:
     collection_interval: 10s
+    root_path: {{ .Values.isWindows | ternary "C:\\hostfs" "/hostfs" }}
```

Comment:
I suppose one can't change from the two values above.

Reply:
no they're mounted as such elsewhere.